lib/tesla/middleware/logger.ex

```diff
@@ -54,7 +54,7 @@ defmodule Tesla.Middleware.Logger do
 
   ### Options
   - `:log_level` - custom function for calculating log level (see below)
-
+  - `:filter_headers` - sanitizes sensitive headers before logging in debug mode
```

Comment:
I'm thinking about the naming: the option is named `filter...` but then in the description it uses `sanitize` - maybe the option should be just named `sanitize_headers` ?

Reply:
i had `sanitize_headers` before but decided to go with `filter_headers` because that's what phoenix and rails also call it.